lib/tasks/scripts/move_user_recorded_data_to_registration_facility.rb

```diff
@@ -9,17 +9,13 @@ def initialize(user, source_facility, destination_facility)
   end
 
   def fix_patient_data
+    patients = Patient.where(registration_user: user, registration_facility: source_facility)
+    fix_pbi_metadata(patients)
```

Comment:
Should we call `fix_pbi_metadata` from the rake task itself instead of coupling it with `fix_patient_data`?

Reply:
No